### PR TITLE
examples: cleanup README.md

### PR DESCRIPTION
**Pull request by chalin**
- Drop outdated text
- Retain what is unique to this page
- Refer reader to the quick start page for the rest

Contributes to https://github.com/grpc/grpc.io/issues/180

@dfawley /cc @ejona86 @jtattermusch

### CODE REVIEW

**Comment by chalin** [comment]
This PR has two commits for two variants of changes to this README -- you can view each commit separately.

The second commit drops the "set PATH" step, step 1.
**Let me know which variant you prefer**.

(I'm trying to keep these instructions as simple as possible, referring the adventurous developer to the [Quick Start](https://grpc.io/docs/languages/go/quickstart/) page for extra information 😄)

File: examples/README.md [hunk 1]
```diff
@@ -1,64 +1,29 @@
...
 
-PREREQUISITES
--------------
+ 1. Get the code:
 
-- This requires Go 1.9 or later
-- Requires that [GOPATH is set](https://golang.org/doc/code.html#GOPATH)
+    ```console
+    $ go get -u google.golang.org/grpc/examples/helloworld/greeter_client
+    $ go get -u google.golang.org/grpc/examples/helloworld/greeter_server
```
Comment (dfawley) [inline examples/README.md:9]: I don't think `-u` is recommended anymore, with modules it has a different behavior than before.

Comment (chalin) [reply examples/README.md]: Done: dropped `-u` flags.

**Comment by chalin** [comment]
Thanks @dfawley. Anything left to do before this is merged?

**Pull request merged**